### ISSUE TITLE
Import CRD docs into new site

### ISSUE THREAD

**Issue by dholbach** [fluxcd/website#115]
We should potentially borrow https://github.com/fluxcd/flux2/blob/main/.github/workflows/docs.yaml and plug it into our `Makefile` somewhere, so we have all relevant CRD docs available too.

Potentially check if we need to add frontmatter?

**Comment by dholbach** [comment]
```
3:54:23 PM: Error: Error building site: "/opt/build/repo/content/en/docs/components/helm/api.md:1:1": plain HTML documents not supported
```

hugo is not pleased.

**Comment by dholbach** [comment]
Ok, issue is resolved.

**Comment by dholbach** [comment]
This was fixed as part of #130
